stove_heating_ai/rootfs/app/src/ml/model.py

```
@@ -0,0 +1,172 @@
+
+    missing_columns = set(FEATURE_COLUMNS) - set(df.columns)
+    if missing_columns:
+        logger.error("Missing required columns: %s", missing_columns)
+        return None, None
+
+    X = df[feature_columns]
+    y = df["Y"]
+
+    from sklearn.model_selection import KFold
+    kfold = KFold(n_splits=5, shuffle=True, random_state=4765)
+    cv_scores = []
```

Ruff (0.8.2)

Local variable `cv_scores` is assigned to but never used

Remove assignment to unused variable `cv_scores`

(F841)